ports/esp32/boards/sdkconfig.base

```diff
@@ -49,6 +49,9 @@ CONFIG_LWIP_PPP_CHAP_SUPPORT=y
 # Use 4kiB output buffer instead of default 16kiB
 CONFIG_MBEDTLS_ASYMMETRIC_CONTENT_LEN=y
 
+# Allow mbedTLS to allocate from PSRAM or internal memory
+CONFIG_MBEDTLS_DEFAULT_MEM_ALLOC=y
```

Comment:
I'd add a short comment on why this is safe (summarise what you wrote in the git commit message for this part).